contracts/cw-proposal-single/src/tests.rs

```diff
@@ -4196,3 +4238,57 @@ fn test_return_deposit_to_dao_on_proposal_failure() {
     // Deposit should now belong to the DAO.
     assert_eq!(balance.balance, Uint128::new(1));
 }
+
+#[test]
+fn test_migrate_from_beta() {
```

Comment:
Same as other PR - might be nice to make this an integration test. No biggie if not though.